tests/setup-runtime-dependencies.sh

```diff
@@ -109,10 +109,17 @@ if [ ! -e $libInstallDir ]; then
     mkdir -p $libInstallDir
 fi
 
+# Query runtime Id
+rid=`$dotnetCmd --version | grep 'Runtime Id:' | sed 's/^ *Runtime Id: *//g'`    
```

Comment:
This won't work anymore in later versions of the CLI: https://github.com/dotnet/cli/pull/2035